### DISCUSSION TITLE
UEFI Secureboot, custom signed opencore bootloader/uefi driver

### DISCUSSION THREAD

**Discussion by keatliang2005** [osy86/HaC-Mini#599] [General]
With Windows 11 coming soon, it required TPM 2.0 and UEFI secure boot.

since Hades Canyon have firmware TPM 2.0 and secure boot, there is a catch/gotcha with Opencore bootloader

we need to sign the opencore bootloader in order to work with UEFI secure boot.

is that possible do that with Hades Canyon, installing custom key to load opencore with UEFI secureboot.

reference
https://dortania.github.io/hackintosh/updates/2021/07/05/acidanthera-july.html

**Comment by osy** [comment]
See https://osy.gitbook.io/hac-mini-guide/details/secure-boot
